micrometer-core/src/main/java/io/micronaut/configuration/metrics/aggregator/AbstractMethodTagger.java

```diff
@@ -53,4 +55,9 @@ public abstract class AbstractMethodTagger {
             return Collections.emptyList();
         }
     }
+
+    @Override
+    public int compareTo(AbstractMethodTagger o) {
+        return Integer.compare(this.getOrder(), o.getOrder());
```

Comment:
You don't need to implement Comparable. Please, use [OrderUtil](https://github.com/micronaut-projects/micronaut-core/blob/4.6.x/core/src/main/java/io/micronaut/core/order/OrderUtil.java) instead.

Reply:
Done

Reply:
@sdelamo just checking in on this and if there are any other changes wanted